### DISCUSSION TITLE
peak to gene

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#609] [Questions / Documentation]
@rcorces @jgranja24 
I was wondering if it is important to include "strand" in peak to gene analysis? It seems that right now, peaks on the - strand could be correlated to a gene that is on the + strand, is that correct?

**Comment by rcorces** [comment]
I dont think "strand" is really relevant here as you also suggested. In most (all?) cases with ATAC-seq data, our peak calls arent stranded. Right @jgranja24 ?
